crates/nu-command/tests/commands/parse.rs

```diff
@@ -197,4 +197,21 @@ mod regex {
 
         assert_eq!(actual.out, "table<letter: string, a: string> (stream)")
     }
+
+    #[test]
+    fn parse_does_not_truncate_list_streams() {
+        Playground::setup("parse_test_streaming_1", |dirs, _sandbox| {
+            let actual = nu!(
+                cwd: dirs.test(), pipeline(
+                r#"
+                    [a b c]
+                    | each {|x| $x}
+                    | parse --regex "[ac]"
+                    | length
+                "#
+            ));
+
+            assert_eq!(actual.out, "2");
+        })
+    }
```

Comment:
maybe it would need to be formatted with `cargo fmt --all` but i suggest simplifying the test in the following way :yum: 
```suggestion
    #[test]
    fn parse_does_not_truncate_list_streams() {
        let actual = nu!(pipeline(
            r#"
                [a b c]
                | each {|x| $x}
                | parse --regex "[ac]"
                | length
            "#
        ));

        assert_eq!(actual.out, "2");
    }
```

Reply:
the other tests in that file might be incorrect as well...
this is part of https://github.com/nushell/nushell/issues/8670 :thinking:

Reply:
Good catch!

Reply:
thanks for the change :pray: